### ISSUE TITLE
Doesn't work on Ubuntu 18.04

### ISSUE THREAD

**Issue by rsgrimes** [Embarcadero/DelphiFMX4Python#20]
Just trying to get Simplest.py to run, with no luck. This is what I've tried:

```$ mkdir ~/tmp/delphifmx
$ cd ~/tmp/delphifmx
$ virtualenv -p /usr/bin/python3.8 delphienv
Running virtualenv with interpreter /usr/bin/python3.8
Using base prefix '/usr'
/usr/lib/python3/dist-packages/virtualenv.py:1086: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
New python executable in /home/rsg/tmp/delphifmx/delphienv/bin/python3.8
Also creating executable in /home/rsg/tmp/delphifmx/delphienv/bin/python
Installing setuptools, pkg_resources, pip, wheel...done.

$ source delphienv/bin/activate
(delphienv) $ python --version
Python 3.8.6

(delphienv) $ pip install delphifmx
Collecting delphifmx
  Downloading delphifmx-0.1.48-cp38-cp38-manylinux1_x86_64.whl (10.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 10.1/10.1 MB 1.2 MB/s eta 0:00:00
Installing collected packages: delphifmx
Successfully installed delphifmx-0.1.48

(delphienv) $ python Simplest.py 
Segmentation fault (core dumped)```

Not very interesting. I decided to try and see what was failing by going in to Python and trying the script manually; this is what I get:

```(delphienv) $ python
Python 3.8.6 (default, Oct  6 2020, 03:22:36) 
[GCC 7.5.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from delphifmx import *
Segmentation fault (core dumped)```

Again, not very interesting. At this point, there really is nothing I can think to try, as it can't even import the darn thing. Any ideas?

**Comment by lmbelo** [comment]
How have you installed Python? Unfurtunately the Python version distributed by apt-get is not supported.

**Comment by rsgrimes** [comment]
I have Python 3.8.6, as indicated above, which I probably installed via apt-get, but the "system" python is Python 3.6.9. Do you have a better recommendation for a supported version?

**Comment by lmbelo** [comment]
You can download a Linux compatible distribution directly from python.org or use the instructions contained in the readme of this repository.

**Comment by rsgrimes** [comment]
Excellent! I just tried with a Python 3.7 that I also had installed, and that one worked! I looked at those scripts, and wonder whether I should use one of those (likely the 3.9) or just stick with what works; given I'm just playing right now, I'll probably just use the one I have until/unless I run into problems or become more serious about this.

Perhaps the README.md could point to those scripts for getting started so others don't get bitten again...

Also, I started this today because I got an email about the quick-start tutorial for the DelphiVCL4Linux project; is there something similar for the FMX version?

Thanks for your help!
